parachain/src/chain_spec/testnet_interlay.rs

```diff
@@ -65,14 +65,14 @@ pub fn staging_mainnet_config(benchmarking: bool) -> InterlayChainSpec {
                     vec![]
                 })
                 .collect(),
+                // 5Ec37KSdjSbGKoQN4evLXrZskjc7jxXYrowPHEtH2MzRC7mv (//sudo/1)
+                Some(get_account_id_from_string(
+                    "5Ec37KSdjSbGKoQN4evLXrZskjc7jxXYrowPHEtH2MzRC7mv",
+                )),
                 crate::chain_spec::interlay::PARA_ID.into(),
                 DEFAULT_BITCOIN_CONFIRMATIONS,
             );
 
-            // 5Ec37KSdjSbGKoQN4evLXrZskjc7jxXYrowPHEtH2MzRC7mv (//sudo/1)
-            genesis.sudo.key = Some(get_account_id_from_string(
```

Comment:
not sure why you refactored this but kept the btc_relay config override unchanged, but I guess it doesn't matter a ton